core/test-src/org/pentaho/di/core/ConditionTest.java

```diff
@@ -0,0 +1,36 @@
+/*! ******************************************************************************
+ *
+ * Pentaho Data Integration
+ *
+ * Copyright (C) 2002-2016 by Pentaho : http://www.pentaho.com
+ *
+ *******************************************************************************
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
+
+package org.pentaho.di.core;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+import org.pentaho.di.core.row.RowMeta;
+
+public class ConditionTest extends TestCase {
+  @Test
+  public void testCondition1() throws Exception {
+    Condition condition = new Condition();
+    condition.setNegated( true );
+    assertFalse( condition.evaluate( new RowMeta(), new Object[]{ "test" } ) );
+  }
```

Comment:
@denis8703 I just meant that you can construct a Condition right in your test which will actually hit line 378 in Condition.java.  The test you created doesn't.  
Small thing, but it's good to make sure the test actually covers what we think it does.  Also, this is sort of a personal preference thing, but I like test names which are descriptive-  something like 
  .testNegatedTrueFuncEvaluatesAsFalse()